src/main/java/org/folio/dew/batch/acquisitions/edifact/jobs/SaveToFileStorageTasklet.java

```diff
@@ -28,30 +30,32 @@
 @StepScope
 @Log4j2
 public class SaveToFileStorageTasklet implements Tasklet {
-  private final ObjectMapper ediObjectMapper;
-  private final FTPStorageService ftpStorageService;
-
-  private final RemoteFilesStorage remoteFilesStorage;
 
   private static final String SFTP_PROTOCOL = "sftp://";
-  @Value("#{jobParameters['edifactFileName']}")
-  private String edifactFileName;
+
+  private final ObjectMapper ediObjectMapper;
+  private final FTPStorageService ftpStorageService;
 
   @Override
   @SneakyThrows
   public RepeatStatus execute(StepContribution contribution, ChunkContext chunkContext) {
-    var stepExecution = chunkContext.getStepContext().getStepExecution();
     var jobParameters = chunkContext.getStepContext().getJobParameters();
-    var ediExportConfig = ediObjectMapper.readValue((String)jobParameters.get(EDIFACT_ORDERS_EXPORT), VendorEdiOrdersExportConfig.class);
-    var uploadedFilePath = (String) ExecutionContextUtils.getExecutionVariable(stepExecution, UPLOADED_FILE_PATH);
+    var ediExportConfig = ediObjectMapper.readValue((String) jobParameters.get(EDIFACT_ORDERS_EXPORT), VendorEdiOrdersExportConfig.class);
+    if (ediExportConfig.getIntegrationType() != ORDERING && ediExportConfig.getTransmissionMethod() != FTP) {
```

Comment:
A bit hard to read with inverse conditions. Also having integration type here will lead us to modify source code of tasklet during adding new integration type. If we could not do such inverse configuration on some higher config level - we can leave as it.